spec/feature/sort_users_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe 'Sort users', type: :feature do
+  let!(:first_user) { FactoryBot.create(:user, last_name: 'bbb', first_name: 'cc') }
+  let!(:second_user) { FactoryBot.create(:user, last_name: 'ccc', first_name: 'aa') }
+  let!(:third_user) { FactoryBot.create(:user, last_name: 'aaa', first_name: 'bb') }
+
+  before(:each) do
+    visit new_user_session_path
```

Comment:
move `visit new_user_session_path` to sign_in_user method

Reply:
renamed @VasylYaremkiv